src/Photos/Enums.cs

```diff
@@ -259,6 +259,9 @@ public enum PHAssetResourceType : long {
 		[iOS (13, 0), TV (13, 0)]
 		[MacCatalyst (13, 1)]
 		AdjustmentBaseVideo = 12,
+		[iOS (17, 0), TV (10, 0)]
```

Comment:
Should this be TV 10? Maybe version 17 would be safer @mandel-macaque

Reply:
I agree on that, also is definitely a MacCatalyst 17, not 17.1

Reply:
@tj-devel709 good catch, addressed!